src/instruments/src/PFD/shared/PFDSimvarPublisher.tsx

```diff
@@ -115,6 +115,14 @@ export interface PFDSimvars {
     xtk: number;
     ldevRequestLeft: boolean;
     ldevRequestRight: boolean;
+    baroMode: number;
+    baroUnit: number;
+    baroPressure: number;
```

Comment:
Maybe a good idea to split these into left/right but reading the same A vars so we don't need to edit the PFD code later (just switch the A vars for L vars).